manuscript/chapter2.md

```diff
@@ -1516,20 +1519,18 @@ const Search = ({ value, onChange, children }) => {
 }
 ~~~~~~~~
 
-But you don't need it for now. That's why you can keep the previous version without the block body. When using block bodies, people often tend to do too many things in the function. By leaving the block body out, you can focus on the input and output of your function.
-
-Now you have one lightweight functional stateless component. Once you would need access to its internal component state or lifecycle methods, you would refactor it to an ES6 class component. In addition you saw how JavaScript ES6 can be used in React components to make them more concise and elegant.
+Now you have one lightweight functional stateless component. When we need accss to the internal component state or lifecycle methods, we can refactor them to ES6 class components. When using block bodies, programmers tend to overload their functions, but leaving the block body out lets you focus on the input and output. JavaScript ES6 in React components makes components more readable and elegant.
```

Comment:
"overload" is the wrong term here, because function overloading is something else https://en.wikibooks.org/wiki/Computer_Programming/Function_overloading